### DISCUSSION TITLE
Filtering by message headers in Kafka UI

### DISCUSSION THREAD

**Discussion by mekkanizer** [provectus/kafka-ui#4481] [Q&A] [answered]
Hello, I know I can add multiple k/v filters when browsing topic messages, like so
```
def systemid = value.SystemId
def msg = value.Message
systemid == "5670" && msg == "Unauthorized"
```

Would it be possible to do the same thing to filter by message headers?
Example: I want header with key `commandName` to have certain value or value_s_ (`"Create"`, `"Change"`, `"Cancel"`)

**Comment by Haarolean** [comment, marked as answer]
hi, this repo is not maintained (#4255), and besides, smart filters are quite broken in this repo :) 
Feel free to switch to https://github.com/kafbat/kafka-ui where we (the original team) have replaced groovy filters with CEL syntax (headers are supported)
